CHANGES_NEXT_RELEASE

```diff
@@ -8,4 +8,5 @@ Fix: Changing cygnus port does not work (#1698)
 [cygnus-ngsi] Changed log level for some events (#1769)
 [cygnus-ngsi][PostgisSink, PostGRESQLSink] Support for native Postgres and Postgis types (#1780)
 [cygnus-ngsi][MySQL] Support for native MySQL types (#1780)
-[cygnus-ngsi] Unattach dockerfile from git PR #1801
\ No newline at end of file
+[cygnus-ngsi] Unattach dockerfile from git PR #1801
+[cygnus-ngsi][HDFS] Dissable metadata files creation when they are empty
```

Comment:
```suggestion
[cygnus-ngsi][HDFS] Dissable metadata files creation when they are empty (#1706)
```

Associated issue number should be mentioned whenever is possible.

Reply:
Yeah, about that. I made a mistake about issue number. I thing there is no issue for this task.

Issue #1706 is about a flag in config file.

Can you confirm @manucarrace, please?

Reply:
Fixed in [d116da2](https://github.com/telefonicaid/fiware-cygnus/commit/d116da284c8d0d6766cb4cb5d8fa0caaefc94940)

Reply:
Issue #1706 is out of the scope of this PR

Reply:
CNR entry on [d116da2](https://github.com/telefonicaid/fiware-cygnus/commit/d116da284c8d0d6766cb4cb5d8fa0caaefc94940)  references issue #1805 

NTC.